src/searchModal/advancedSearch.js

```diff
@@ -91,18 +91,21 @@ export default function advancedSearchFactory(config) {
         /**
          * Builds substring of search query with the advanced criteria conditions
          */
-        getAdvancedCriteriaQuery: function () {
+        getAdvancedCriteriaQuery: function (hasSearchInput) {
             const advancedSearchCriteria = _.filter(criteriaState, criterion => criterion.rendered === true);
             let query = '';
 
             advancedSearchCriteria.forEach(renderedCriterion => {
+                if (hasSearchInput || query.trim().length !== 0) {
+                    query += ' AND ';
```

Comment:
In situation when user added many criterias and only in first placed value, will be query like "criteria1:value AND AND AND" ?

Reply:
Yes, If you add a criterion and its empty... Thanks, I will fix this case

Reply:
Done

![image](https://user-images.githubusercontent.com/34692207/105006196-98030280-5a36-11eb-9a54-1c32e9f342b8.png)